pkg/controller/validatingadmissionpolicystatus/controller.go

```diff
@@ -30,13 +30,20 @@ import (
 	admissionregistrationv1beta1apply "k8s.io/client-go/applyconfigurations/admissionregistration/v1beta1"
 	informerv1beta1 "k8s.io/client-go/informers/admissionregistration/v1beta1"
 	admissionregistrationv1beta1 "k8s.io/client-go/kubernetes/typed/admissionregistration/v1beta1"
+	"k8s.io/client-go/openapi"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
+	"k8s.io/kubernetes/pkg/controller/validatingadmissionpolicystatus/schemawatcher"
+	"k8s.io/kubernetes/pkg/controller/validatingadmissionpolicystatus/typetracker"
 )
 
 // ControllerName has "Status" in it to differentiate this controller with the other that runs in API server.
 const ControllerName = "validatingadmissionpolicy-status"
 
+// schemaWatchPollInterval is the interval in which the schema watcher refreshes.
+// 10s result in 6 queries per minute.
```

Comment:
Would you please add the scalability report in the PR so reviewer knows better on how those number come from?

Reply:
Clarification: this number is not from the Scalability test. The number is a magic.
The scalability test concluded that the "reasonably worst" uncached request for the discovery root takes about 13ms measured on the client-side, but surely we cannot do an 100 qps against the discovery root.

Reply:
I agree that we should begin with a larger number and we could always enhance the performance later without affecting existing use cases.
 /cc @jpbetz WDT?

Reply:
I'd be okay with every 30 or 60 seconds for a release to settle this in.  The worst case latency on the client side makes me suspect this is low risk.

@wojtek-t for visibility in case this shows up in scalability.